.github/workflows/copyright-validate.yaml

```diff
@@ -0,0 +1,19 @@
+name: Check Copyright Headers
+
+on:
+  # - pull_request [should we include for all types]
+  push:
+    branches:
+      - gh-copyright-workflow
+jobs:
+  copywrite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: hashicorp/setup-copywrite@32638da2d4e81d56a0764aa1547882fc4d209636 # v1.1.3
+        name: Setup Copywrite
+        with:
+          version: v0.16.4
```

Comment:
Any reason we pin to this version?

Reply:
I noticed this is the version used in other products but we have `version: v0.18.0` mentioned in setup-copywrite [repo](https://github.com/hashicorp/setup-copywrite), I can change to this version, not sure if there's a way to get latest always instead of pinning?

Reply:
I think it will pull the latest if you just don't include the version

Reply:
I removed the version and the checksum verification [step](https://github.com/hashicorp/setup-copywrite?tab=readme-ov-file#faq), which didn't seem necessary in our workflow